apps/omg_child_chain/lib/omg_child_chain/block_queue.ex

```diff
@@ -76,46 +77,48 @@ defmodule OMG.ChildChain.BlockQueue do
   """
   def handle_continue(:setup, args) do
     _ = Logger.info("Starting #{__MODULE__} service.")
-    :ok = Eth.node_ready()
-
-    {:ok, parent_start} = Eth.RootChain.get_root_deployment_height()
+    finality_threshold = Keyword.fetch!(args, :submission_finality_margin)
```

Comment:
block queue init params got cleaned up, so that they're not fetched from the init, if they can be passed into the process